scripts/integration-test-pipelines.sh

```diff
@@ -47,7 +47,14 @@ setup_test_pipeline() {
 
 }
 
+# FIXME: Remove when deployed.
+if $FLY_CMD -t "${FLY_TARGET}" pipelines | grep -qE "^usage-events-collector\s"; then
+  $FLY_CMD -t "${FLY_TARGET}" rename-pipeline \
+    -o usage-events-collector \
+    -n paas-billing
+fi
+
 setup_test_pipeline rds-broker alphagov paas-rds-broker master
 setup_test_pipeline compose-broker alphagov paas-compose-broker master compose-broker-secrets.yml
-setup_test_pipeline usage-events-collector alphagov paas-usage-events-collector master
+setup_test_pipeline paas-billing alphagov paas-billing master
```

Comment:
I think the name of the pipeline should be billing as we usually don't add the `paas-` prefix.

Reply:
I think we should leave it in, like `paas-product-pages` and `paas-tech-docs`, because it's less descriptive without it.

Reply:
ok